### PR TITLE
Update help text for color mapping

### PR DESCRIPTION
**Pull request by clarasb**
This PR:
- updates the help text for the simplified color mapping
- fixes a small typo.

### CODE REVIEW

File: src/resources/lang.json [hunk 1]
```diff
@@ -972,7 +972,7 @@
     },
     {
       "en": "Stepwise color mapping where values are bounds of value ranges mapped to the same color",
-      "de": "Schrittweise Farbzuordnung, bei der die Werte Bereichsgrenzen darstellen, die einer einzelnen Frabe zugeordnet werden",
+      "de": "Schrittweise Farbzuordnung, bei der die Werte Bereichsgrenzen darstellen, die einer einzelnen Farbe zugeordnet werden",
```
Comment (forman) [inline src/resources/lang.json:975]: Die schwedische Übersetzung kommt mir zu kurz vor. Ich fixe das.

Comment (forman) [reply src/resources/lang.json:975]: Oops, german :)

**Review by forman — APPROVED** [review]

File: public/docs/color-mappings.en.md [hunk 2]
```diff
@@ -16,13 +16,14 @@ such as `#FFA500CD`.
...
-  `<color>` associated with the first `<value>` of the boundary.
-  The last color value is therefore ignored.
-* Type **Key**: The values are integer keys that directly identify 
-  the associated color. Should be used for data of type integer.
-  Together with a categorical normalisation **CAT**, 
-  this color mapping type will also allow for a categorical map legend. 
+* **Continuous:** Continuous color assignment, where each `<value>` 
+  represents a support point of a color gradient.
+* **Stepwise:** Stepwise color mapping where values are bounds of value 
+  ranges mapped to the same color. A `<color>` gets associated with the
```
Comment (konstntokas) [inline public/docs/color-mappings.en.md:22]:
```suggestion
* **Stepwise:** Stepwise color mapping where values within the range of two
  subsequent `<value>`s are mapped to the same color. A `<color>` gets associated with the 
```

**Review by konstntokas — APPROVED** [review]
Looks good. Just a few suggestions to enhance readability.

File: public/docs/color-mappings.en.md [hunk 3]
```diff
@@ -16,13 +16,14 @@ such as `#FFA500CD`.
...
-  values or _nodes_.
-* Type **Bound**: Adjacent values form value _bounds_ which map to the 
-  `<color>` associated with the first `<value>` of the boundary.
-  The last color value is therefore ignored.
-* Type **Key**: The values are integer keys that directly identify 
-  the associated color. Should be used for data of type integer.
-  Together with a categorical normalisation **CAT**, 
-  this color mapping type will also allow for a categorical map legend. 
+* **Continuous:** Continuous color assignment, where each `<value>` 
+  represents a support point of a color gradient.
```
[the remark this answers is not in the document]
Comment (forman) [reply public/docs/color-mappings.en.md:20]: Grid is more likely in 2+D context.

**Pull request merged**